app/(chat)/integrations/vercel/callback/page.tsx

```
@@ -1,47 +1,67 @@
+    if (!code) return;
+
+    const fetchToken = async () => {
+      try {
+        const res = await fetch('https://api.vercel.com/v2/oauth/access_token', {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/x-www-form-urlencoded',
+          },
+          body: new URLSearchParams({
+            code,
+            client_id: process.env.VERCEL_CLIENT_ID!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)